src/main/java/agh/edu/pl/repackaging/classes/OpenTelemetryClasses.java

```diff
@@ -77,14 +75,14 @@ public void addOpenTelemetryFolders() throws IOException {
         if ((entry.getName().startsWith("inst/") || entry.getName().startsWith("io/"))
             && !entry.isDirectory()) {
           if (entry.getName().endsWith(".classdata")) {
```

Comment:
Consider dividing this method into smaller ones

Reply:
Yes, and maybe we can generalize first two branches of that `if`? They look quite similar, maybe this can be extracted to other method